redbot/core/bot.py

```diff
@@ -35,6 +35,7 @@
 __all__ = ["RedBase", "Red", "ExitCodes"]
 
 NotMessage = namedtuple("NotMessage", "guild")
+T_BIC = TypeVar("T_BIC")
```

Comment:
Would this work as a type hint or is there something I'm missing?
```suggestion
T_BIC = TypeVar("T_BIC", Callable[[commands.Context], Awaitable[Any]])
```